feature-group-definitions/slot-assign.dist.yml

```
@@ -0,0 +1,23 @@
+# Generated from: slot-assign.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Imperative slot assignment
+description: The `assign()` method for `<slot>` elements assigns the given nodes to the slot. This allow you to control slotting using Javascript, as an alternative to using the `slot` and `name` HTML attributes. The nodes must be direct children of a shadow host and the shadow root must be created with the `slotAssignment `parameter set to "manual". Also known as manual slot assignment.
```

> Error: description in slot-assign.yml is too long, 377 characters. The maximum allowed length is 300.

Trying to jettison the least essential, this is 302 characters and would pass if we counted plaintext length:

```suggestion
description: The `assign()` method for `<slot>` elements assigns nodes to the slot. This is an alternative to using the `slot` and `name` HTML attributes. The nodes must be children of a shadow host and the shadow root must be created with the `slotAssignment` set to "manual". Also known as manual slot assignment.
```

---

```suggestion
description: The `assign()` method for `<slot>` elements assigns nodes to the slot, as an alternative to using the `slot` and `name` HTML attributes. The nodes must be children of a shadow host and the shadow root must be created with the `slotAssignment` set to "manual". Also known as manual slot assignment.
```

NB: this is the dist file, you'll want to apply it to the authored YAML.